package.json

```diff
@@ -31,6 +31,7 @@
     "eslint": "^2.9.0",
     "@ljharb/eslint-config": "^4.0.0",
     "parallelshell": "^2.0.0",
+    "iconv": "^2.2.0",
```

Comment:
Yes, there is easily another way. I wanted to provide an example that actually shows how to implement a ShiftJis converter (might be a reoccurring issue)